app/services/katello/pulp3/content_view_version/import.rb

```diff
@@ -45,14 +45,67 @@ def destroy_importer(importer_href)
         end
 
         class << self
-          def check_permissions!(path)
+          def check!(content_view:, metadata:, path:)
+            check_permissions!(metadata: metadata, path: path)
+            check_metadata!(content_view: content_view, metadata: metadata)
+          end
+
+          # rubocop:disable Metrics/MethodLength
```

Comment:
Can this method be split into multiple methods?  The benefit would be that you could name them so it's clearer what's happening, like
```
check_cvv_does_not_already_exist
check_from_cvv_is_present
etc.
```
also it would avoid having to disable this cop.